Formula/pdftilecut.rb

```diff
@@ -0,0 +1,23 @@
+class Pdftilecut < Formula
+  desc "Sub-divide a PDF page(s) into smaller pages so you can print them"
+  homepage "https://github.com/oxplot/pdftilecut"
+  url "git@github.com:oxplot/pdftilecut.git", using: :git, tag: "v0.4", revision: "af9ca3bff60eed47026a6e1c40b6ef85e5b36393"
```

Comment:
```suggestion
  url "https://github.com/oxplot/pdftilecut.git",
      tag: "v0.4",
      revision: "af9ca3bff60eed47026a6e1c40b6ef85e5b36393"
```
Mind the spacing. Is there any reason you're not using the release tarball?

Reply:
Thanks. The tarball does not contain the git submodules required to build the binary, but maybe building is not mandatory ?

Reply:
Ah, I see. Do you mean the submodules here? https://github.com/oxplot/pdftilecut/tree/master/c-deps

We'd prefer if you use Homebrew's `jpeg-turbo` and `qpdf` to build this. macOS also provides a copy of `zlib`, though there's also a Homebrew formula if the built-in one doesn't suit for some reason.

Reply:
It seems the spacing failed the syntax check. Did I missed something ?

Reply:
I'll try to use the Homebrew's favorite dependencies instead, but I'm not sure I will be able to achieve this: I'm not the command author

Reply:
Usually, there's a way to substitute dependencies for those of your own. For example, you might already have `jpeg-turbo` or `qpdf` installed, and don't want `pdftilecut` to install another one.

If instructions on how to do this aren't in the docs I guess you can ask the author.

Reply:
I tried to use the binary release instead of source and it works.
Is this a good practice ?

Reply:
No, that's not acceptable, unfortunately: https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

Reply:
Ok, got it !